tests/rule_based_profiler/parameter_builder/test_numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -237,3 +244,218 @@ def test_oneshot_numeric_metric_range_multi_batch_parameter_builder_bobby(
     # if false positive rate is higher, our range should be more narrow
     assert actual_value_01_lower < actual_value_05_lower
     assert actual_value_01_upper > actual_value_05_upper
+
+
+def test_bootstrap_numeric_metric_range_multi_batch_parameter_builder_bobby_false_positive_rate_one(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    data_context: DataContext = (
+        bobby_columnar_table_multi_batch_deterministic_data_context
+    )
+
+    # BatchRequest yielding three batches
+    batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder
+    numeric_metric_range_parameter_builder = (
+        NumericMetricRangeMultiBatchParameterBuilder(
+            name="row_count_range",
+            metric_name="table.row_count",
+            estimator="bootstrap",
+            false_positive_rate=1.0,
+            round_decimals=0,
+            data_context=data_context,
+        )
+    )
+
+    variables: Optional[ParameterContainer] = None
+
+    domain: Domain = Domain(
+        domain_type=MetricDomainTypes.TABLE,
+    )
+    parameter_container: ParameterContainer = ParameterContainer(parameter_nodes=None)
+    parameters: Dict[str, ParameterContainer] = {
+        domain.id: parameter_container,
+    }
+
+    assert parameter_container.parameter_nodes is None
+
+    error_message: str = re.escape(
+        f"""You have chosen a false_positive_rate of 1.0, which is too close to 1.
+A false_positive_rate of {1 - NP_EPSILON} has been selected instead."""
+    )
+
+    with pytest.warns(UserWarning, match=error_message):
+        numeric_metric_range_parameter_builder.build_parameters(
+            domain=domain,
+            variables=variables,
+            parameters=parameters,
+            batch_request=batch_request,
+        )
+
+
+def test_bootstrap_numeric_metric_range_multi_batch_parameter_builder_bobby_false_positive_rate_negative(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    data_context: DataContext = (
+        bobby_columnar_table_multi_batch_deterministic_data_context
+    )
+
+    # BatchRequest yielding three batches
+    batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder
```

Comment:
@NathanFarmer We do not need this declaration -- it was already made above.  Thanks.

Reply:
@alexsherstinsky it was complaining that the line was too long

Reply:
@alexsherstinsky I have reverted the change